src/cli/analyzer-cli-config.ts

```diff
@@ -30,4 +31,8 @@ export interface AnalyzerCliConfig {
 
 	ts?: typeof tsModule;
 	cwd?: string;
+	pathAsAbsolute?: boolean;
```

Comment:
this is only used in the web types stuff it seems. any particular reason it shouldn't just be part of the web types config in that case?

Reply:
It's because `WebTypesTransformerConfig` is only holding some static text values that are injected as is in produced result, kind of a template.
But I get your point, I'll do the update, it's not an issue if it hold some configuration too, moreover with lastly added package.json configuration support.